logger/migrations/0000_init.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS logs (
+    deployment_id TEXT,        -- The deployment that this log line pertains to.
+    shuttle_service_name TEXT, -- The shuttle service which created this log.
```

Comment:
Oh, this is the name of the service inside shuttle `deployer`, `builder`, `resource_recorder`, etc, that generated this log. This will allow us to get access to the resource logs and other things

Reply:
Hmm.. I think we're still exporting those logs to honeycomb. Are we planning on exporting them to the logger service?

Reply:
Clarified this internally, the answer is yes. We want to send logs relating to a service that are relevant for a specific user deployment too.